packages/accordion-react/src/AccordionItem.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { ReactNode, useState } from "react";
+/* There are no types for core-toggle, so we have to use ts-ignore
+Hopefully someone (us?) will write types for it sometime soon */
+// @ts-ignore
+import CoreToggle from "@nrk/core-toggle/jsx";
+
+interface Props {
+    title: string;
+    children: ReactNode;
+    startExpanded?: boolean;
+}
+
+export function AccordionItem({ children, title, startExpanded = false }: Props) {
+    const [isOpen, setIsOpen] = useState(startExpanded);
+    const openClassName = isOpen ? " jkl-accordion-item--expanded" : "";
+    const onToggle = () => setIsOpen(!isOpen);
+    return (
+        <div className={`jkl-accordion-item${openClassName}`}>
+            <button className="jkl-accordion-item__title">
```

Comment:
What does the button actually do here? I might be misunderstanding this, but there is no eventHandler on the button, so should this not be a div or span or something other than a button?

Reply:
`type="button"` to the rescue!